prism.sh

```diff
@@ -3,4 +3,32 @@ set -e
 
 cd examples/prism
 docker-compose build
-docker-compose up --force-recreate --abort-on-container-exit --remove-orphans
+docker-compose up -d --force-recreate --remove-orphans
```

Comment:
I think it would be much simpler to do an `up` for each testing container rather than run them all in parallel and need to explicitly check that each is complete. We wouldn't get parallel runs from the testers so a bit slower, but less complex checks here. For example:
```
docker-compose up --force-recreate --abort-on-container-exit --remove-orphans -d go-client-test
docker-compose up --force-recreate --abort-on-container-exit --remove-orphans -d java-test
```
What do you think?

Reply:
I'll optimise this in [DII-351](https://issues.corp.twilio.com/browse/DII-351).